### PR TITLE
[TW-484] Limitations for multi protocol documentation

### PR DESCRIPTION
**Pull request by mark-anthony-postman**
- Note limitations when creating documentation for collections with gRPC and WebSocket requests
- Fix vale style errors

### CODE REVIEW

File: src/pages/docs/publishing-your-api/authoring-your-documentation.md [hunk 1]
```diff
@@ -59,7 +59,10 @@ Use descriptions to tell people who use your collection more about what your col
 To add or edit the description for an existing collection, folder, or request:
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/publishing-your-api/authoring-your-documentation.md:59]: In line 57, can you please remove the extra "of" from "the purpose of each of request"? Thanks!

File: src/pages/docs/sending-requests/websocket/websocket.md [hunk 2]
```diff
@@ -184,12 +179,12 @@ The following settings can be configured for your WebSocket request:
...
 |-----|-----|
-| Client version  | The Socket.IO client version to be used to connect with the server. (Socket.IO only)|
-| Handshake path | The server-side path that will be captured. (Socket.IO only)  |
+| Client version  | The Socket.IO client version to be used to connect with the server. (Socket.IO)|
+| Handshake path | The server-side path that will be captured. (Socket.IO)  |
 | Handshake request timeout | How long the handshake request will wait before timing out, in milliseconds. This is reset after every redirection. |
 | Reconnection attempts | The maximum number of reconnection attempts before disconnecting. |
 | Reconnection intervals | The period in milliseconds between reconnection attempts. |
-| Maximum message size | The maximum allowed message size, in megabytes. To receive messages of any size, set this to 0. (Raw WebSocket only)|
+| Maximum message size | The maximum allowed message size, in megabytes. To receive messages of any size, set this to 0. (Raw WebSocket)|
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/sending-requests/websocket/websocket.md:187]: I'm wondering if the use of "only" in this context is an edge case. It's my understanding that there are multiple types of WebSocket connections, and each row that specifies a type of connection only applies to that type. I think it's clearer to use "only", which clarifies that a row only applies to a specific type.

What do you think?

**Review by MatthewPinsker-postman — APPROVED** [review]
Looks good to me! I added a few questions/thoughts.

**Review by petewitcher — APPROVED** [review]
LGTM!

File: src/pages/docs/sending-requests/websocket/websocket.md (hunk 2 above)
Comment (mark-anthony-postman) [reply src/pages/docs/sending-requests/websocket/websocket.md:187]: I think you're right, I was letting vale make me overzealous 😆

**Review by jkonrath-postman — APPROVED** [review]
LGTM!

**Review by kamaraju-p — APPROVED** [review]
LGTM! ✅

**Review by SterlingChin — APPROVED** [review]

**Pull request merged**